apps/web/core/components/common/skeleton/team-member-header-skeleton.tsx

```
@@ -0,0 +1,124 @@
+				</div>
+			</div>
+		</div>
+	);
+
+	const renderHeaderSkeleton = () => {
+		switch (view) {
+			case IssuesView.TABLE:
+				return renderTableHeaderSkeleton();
+			case IssuesView.BLOCKS:
+				return renderBlockHeaderSkeleton();
+			case IssuesView.CARDS:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)